src/components/Button/src/Button.vue

```diff
@@ -104,6 +98,33 @@ function setColorVariables(tokens, variant) {
 	};
 }
 
+const TEXT_STYLES = new Set([
+	'fontFamily',
+	'fontWeight',
+	'fontSize',
+	'fontStyle',
+	'textTransform',
+	'textDecoration',
+]);
+
+function isValidCss(style, value) {
+	if (global.CSS) {
+		return global.CSS.supports(style, value);
+	}
+	return false;
```

Comment:
`global.css` is undefined when server-side rendering, so this will always return `false` in that use-case, which might break things or just annoy people, lol. I think defaulting to `true` is the lesser of both evils, assuming you haven't already tested this with SSR and it works fine.